technical_guides/css.md

```diff
@@ -10,7 +10,7 @@ Store all colors, font families, font weights, and font sizes in variables.
 
 Use variable-heavy math. Avoid magic numbers whenever possible (and comment when necessary). Store all variables in a separate file so you can see where it's possible to consolidate.
 
-You might consider using [Scut](https://davidtheclark.github.io/scut/) for common utilities and tricks.
+You might consider using [Scut](https://ramseyinhouse.github.io/scut) for common utilities and tricks.
```

Comment:
Is this still relevant?  @JeffreyATW thoughts?

Reply:
It is not.

Reply:
dropped this link, left the other scut link - feel free to re-work that guide